packages/showcases/css/stories/components/text-input/examples/default.html

```diff
@@ -0,0 +1,10 @@
+<div class="block">
```

Comment:
I'm not a huge fan of "block". According to me it's generic and maybe that's the objective ?
I would prefer something like "text-input-wrapper" or "text-input-box", what do you think about that ?
(If you change take care to change it in the other files ;) )

Reply:
This class is global for storybook and add a padding between each block. But yes I will use a tailwind css utility classes instead in every file I think.

Reply:
Oh that's true sorry, look good to me too ;)